### PR TITLE
Update include_multiple_projects_version and fixes around remote revision

### PR DESCRIPTION
**Pull request by massonju**
This PR contains several updates/fixes:
- include_multiple_projects_version: update expected_version
- return default value for __get_remote_[url|revision]
- check remote revision when getting project branch

### CODE REVIEW

**Comment by massonju** [comment]
Request review from @david-baylibre

File: repo_resource/common.py [hunk 1]
```diff
@@ -267,10 +267,10 @@ def __add_remote_revision(self, remote, rev):
         self.__remote_revision[remote] = rev
 
     def __get_remote_url(self, remote):
-        return self.__remote_url[remote]
+        return self.__remote_url.get(remote, None)
```
Comment (david-baylibre) [inline repo_resource/common.py:270]: Default value is None if value isn't found in Dict, so no need for the `None` parameter:
https://docs.python.org/3/library/stdtypes.html#dict.get

I don't know Google repo's behaviour if no remote is defined for the project in the manifest. In any case, we use this in the check step only, to compute the manifest version, so it's fine with me.

File: repo_resource/common.py [hunk 2]
```diff
@@ -267,10 +267,10 @@ def __add_remote_revision(self, remote, rev):
...
 
     def __get_remote_revision(self, remote):
-        return self.__remote_revision[remote]
+        return self.__remote_revision.get(remote, None)
```
Comment (david-baylibre) [inline repo_resource/common.py:273]: Same thing with default `None`

**Review by david-baylibre — CHANGES_REQUESTED** [review]
Thanks @massonju for the updates

Just a small comment on dict.get()

**Review by david-baylibre — APPROVED** [review]

**Review by makohoek — APPROVED** [review]

**Pull request merged**